Makefile

```diff
@@ -22,6 +22,8 @@ push-preview:
 
 install-drupal:
 	vendor/bin/drush site-install prisoner_content_hub_profile --existing-config -y
+	vendor/bin/drush pm-uninstall prisoner_hub_entity_access
+	vendor/bin/drush pm-enable prisoner_hub_prison_access
```

Comment:
Installing the new module manually (not via config) so that the new tests will pass on circleCI.
These two lines can be removed after this has been deployed.